README.md

```diff
@@ -148,7 +148,7 @@ Description=Run winetricks update script
 After=network.target
 
 [Service]
-ExecStart=/usr/bin/update_winetricks
+ExecStart=/bin/bash /usr/bin/update_winetricks
```

Comment:
This should be /bin/sh, not /bin/bash:
https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L1